content/python/concepts/sets/terms/copy/copy.md

```diff
@@ -0,0 +1,63 @@
+---
+Title: '.copy()'
+Description: 'Returns a copy of a specified set.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Collections'
+  - 'Functions'
+  - 'Methods'
+  - 'Sets'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.copy()`** method returns a copy of an original set. It references the elements in the original set and can be used when it is important to work with the new set and not modify the original set.
+
+## Syntax
+
+```pseudo
+set.copy()
+```
+
+- The `.copy()` method has no arguments.
+
+## Example
+
+This example creates a shallow copy of a simple set. 
+
+```py
+original_set = { 'First', 'Second', 'Third' }
+print (original_set)
+
+new_set = original_set.copy()
+new_set.add('Fourth')
+print(new_set)
+```
+
+This would output the result:
+
+```shell
+# original_set
+{'First', 'Second', 'Third'}
+
+# new_set
+{'First', 'Second', 'Third', 'Fourth'}
+```
+
+Note: Unlike lists, a set element has no order or index. The ouput of this result is shown in order, but the result can be different. 
```

Comment:
Modifications:

- Let's use the '>' symbol to indicate a 'Note'
- Let's link 'lists' to its doc entry
- Let's change the text a bit

```suggestion
> Note: Unlike [lists](https://www.codecademy.com/resources/docs/python/lists), a set element has no order or index. Hence, though the elements in the output are shown in order, they can be placed differently in other cases.
```